source/developers/concepts/platform_overview.rst

```diff
@@ -0,0 +1,84 @@
+Platform Overview
+#################
+
+The Open edX Platform is made up of multiple web services that work together to
+provide all the tools you might need to build and run courses online.
+
+Tech Stack
+**********
+
+While there are a few exceptions, for the most part the technology stack for the
```

Comment:
Either "While there are a few exceptions" or "for the most part" can be removed and the sentiment withstands.